source/compiler/aslresources.y

```diff
@@ -214,6 +214,7 @@ ResourceMacroTerm
     | Memory32Term                  {}
     | PinConfigTerm                 {}
     | PinFunctionTerm               {}
+    | ClockInputTerm                {}
```

Comment:
I'm wondering if the (mis)location is deliberate (to split Pin*() resources instead of putting alphabetically).